### PR TITLE
[v2.6] Added ACI-CNI 5.2.7.1 templates

### PR DESCRIPTION
**Pull request by LavanyaVaddavalli**
https://github.com/rancher/rancher/issues/41761

### CODE REVIEW

File: rke/k8s_rke_system_images.go [hunk 1]
```diff
@@ -8418,6 +8418,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		// Enabled in v2.6.12
```
Comment (snasovich) [inline rke/k8s_rke_system_images.go]:
```suggestion
		// Enabled out of band post v2.6.13
```

File: rke/k8s_rke_system_images.go [hunk 2]
```diff
@@ -8770,6 +8814,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		// Enabled in v2.6.12
```
Comment (snasovich) [inline rke/k8s_rke_system_images.go]:
```suggestion
		// Enabled out of band post v2.6.13
```

**Review by snasovich — DISMISSED** [review]
Confirmed new versions added only bump ACI image versions. Couple comments to fix; otherwise LGTM.
Approving provided the comments are fixed.

File: rke/k8s_rke_system_images.go (hunk 2 above)
Comment (LavanyaVaddavalli) [reply rke/k8s_rke_system_images.go]: Addressed 34ac11f02f25a2224375ece63d51309aa44ce084

File: rke/k8s_rke_system_images.go (hunk 1 above)
Comment (LavanyaVaddavalli) [reply rke/k8s_rke_system_images.go]: Addressed 34ac11f02f25a2224375ece63d51309aa44ce084

**Comment by LavanyaVaddavalli** [comment]
Testing is under progress

**Review by snasovich — DISMISSED** [review]

**Comment by LavanyaVaddavalli** [comment]
@snasovich addressed comments in ddffe3b1c48a808b8f2275736834fe913e3186a8

**Review by snasovich — DISMISSED** [review]
Approved, but also hinging on RKE1 PR https://github.com/rancher/rke/pull/3254

**Review by snasovich — APPROVED** [review]

**Review by kinarashah — APPROVED** [review]

**Pull request merged**